packages/fx-core/src/plugins/resource/bot/plugin.ts

```diff
@@ -789,18 +791,15 @@ export class TeamsBotImpl {
       );
       botAuthCreds = await AADRegistration.registerAADAppAndGetSecretByAppStudio(
         appStudioToken!,
-        aadDisplayName
+        aadDisplayName,
+        this.config.scaffold.objectId,
+        this.config.scaffold.botId
       );
 
-      if (!this.config.scaffold.botId) {
```

Comment:
Good to refactor it as single of truth. If we refactor this code of L795, do we need to also refactor for that in local debug of L764?